recipes/django-leaflet/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "django-leaflet" %}
+{% set version = "0.22.0" %}
+{% set sha256 = "2a1f4f31bff2b3a58158372f7910053670483e3e11f8ac365ce75f3e3a4ac11c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ordereddict  # [py<27]
+    - sphinx
+    - sphinx-autobuild
+
+test:
```

Comment:
I'm not sure if you need to do something like the following to test this package or if the simple import is sufficient:

https://github.com/conda-forge/staged-recipes/pull/3793/files#diff-115913f45ac9f5f7ceef86959e275086R30

Reply:
it seems exactly what I am looking for :)